src/odemis/acq/stream/_base.py

```diff
@@ -1358,3 +1359,9 @@ def getRawMetadata(self):
         A list of metadata dicts is returned.
         """
         return [None if data is None else data.metadata for data in self.raw]
+
+    def invalidate_cache(self):
```

Comment:
I've changed it to be the client responsibility, and renamed it to be 'force_image_update', because not all of the streams/projections have caches to invalidate. It works, but i need to update .rect for the tiledshadows so that fit_to_content works correctly, but this is proving more difficult because changing it changes the tile-read calculation, and it doesn't let me set it outside the current range (which is staticly calculated on load). Alternatively, it might be better to change how getBoundingBox works for RGBSpatialProjection, and just use the image content instead of the .rect.range?

Reply:
fixed the bounding box issue, now it's recalculated rather than fixed at .rect